spec/models/apartment_spec.rb

```diff
@@ -0,0 +1,123 @@
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', price: 'unlimited', pets: '2 small dogs.', bedrooms: 7, bathrooms: 7, image: "http//facebook1234"
+
+  expect(apartment.errors[:email]).to_not be_empty
+  end
+
+it 'is not valid without a price' do
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', pets: '2 small dogs.', bedrooms: '7', bathrooms: '7', image: "http//facebook1234"
+
+  expect(apartment.errors[:price]).to_not be_empty
+end
+
+it 'is not valid without a pets' do
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', bedrooms: 7, bathrooms: 7, image: "http//facebook1234"
+
+  expect(apartment.errors[:pets]).to_not be_empty
+end
+
+it 'is not valid without a bedrooms' do
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs', bathrooms: 7, image: "http//facebook1234"
+
+  expect(apartment.errors[:bedrooms]).to_not be_empty
+end
+
+it 'is not valid without a bathrooms' do
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs', bedrooms: 7, image: "http//facebook1234"
+
+  expect(apartment.errors[:bathrooms]).to_not be_empty
+end
+
+it 'is not valid without a image' do
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs', bedrooms: 7, bathrooms: 7
+
+  expect(apartment.errors[:image]).to_not be_empty
+end
+
+it 'is not valid without a user_id' do
+
+  apartment = user.apartments.create street: 'Canal ST', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs', bedrooms: 7, bathrooms: 7, image: "http//facebook12345"
+
+ expect(apartment.errors[:user_id]).to be_empty
+   end
+end
+
+
```

Comment:
no need to create the user in every it statement.  You defined it in the describe and that is all you need.